spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/pubsub/GcpPubSubProperties.java

```diff
@@ -126,6 +126,11 @@ public void setKeepAliveIntervalMinutes(int keepAliveIntervalMinutes) {
 		 */
 		private final Batching batching = new Batching();
 
+		/**
+		 * Enable message ordering setting.
+		 */
+		private Boolean enableMessageOrdering;
```

Comment:
why not a primitive boolean? Default state is false; would be no need to mess with nulls.

Reply:
+1 yeah would also prefer the primitive boolean to avoid implying that not setting it is different than setting it to `false`.

Reply:
Sometimes we follow the pattern that is to allow not setting it, so that the client library default can be used automatically. Otherwise, we have to maintain the default, which in this case probably doesn't matter.